ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py

```diff
@@ -14,14 +14,16 @@
 # You should have received a copy of the GNU General Public License along
 # with this program. If not, see <http://www.gnu.org/licenses/>.
 
+from operator import truediv
 import os
 import sys
 import argparse
 import subprocess
 from datetime import date
 
-PYTHON_HEADERS = ["#!/usr/bin/env python", "#!/usr/bin/python"]
-ACCEPTED_EXTENSIONS = ["py", "c", "h", "cpp", "hpp"]
+# Include bash in python files as we also put licences in bash files too.
+PYTHON_HEADERS = ["#!/usr/bin/env python", "#!/usr/bin/python", "#!/bin/bash", "#!/usr/bin/env bash"]
+ACCEPTED_EXTENSIONS = ["py", "c", "h", "cpp", "hpp", "msg", "yml", "yaml", "sh", "xml", "xacro", "dae", "launch"]
```

Comment:
I assume that msg extension is for ROS messages? Shouldn't we also have then .srv and .action added?

Reply:
I guess thats true, I just based this off the documentation I got :) do .srv and .action use # based comments?

Reply:
I would say its the same 'type' as .msg so if we are adding licenses in the messages then the same should happen in .srv and .action